__tests__/review.test.js

```diff
@@ -148,4 +155,379 @@ describe('ReviewComp testing', () => {
+    let comps = wrapper.find('ReviewComp');
+    let moreReviewBtn = wrapper.find('.btn').at(1);
+    expect(moreReviewBtn.text()).toContain('MORE REVIEWS');
+    moreReviewBtn.simulate('click');
+    moreReviewBtn = wrapper.find('.btn').at(1);
+    moreReviewBtn.simulate('click');
+    moreReviewBtn = wrapper.find('.btn').at(1);
+    expect(moreReviewBtn).toHaveLength(0);
+  });
+
+});
+
+// Sort options
+
+// Sort options should have Helpful, latest, Relevance (done)
+
+// Can sort helpful properly (done)
+
+// Can sort Newest properly (done)
+
+// Can sort Relevance properly (done)
+
+// By default, the option should be in order of relevance (done)
+
+describe('Sort options testings', () => {
+  test('Sort options should have Helpful, latest, Relevance, By default, the option should be in order of relevance', () => {
+    let filters = [];
+    let wrapper = shallow(<ReviewSort onChange={() => { }} />);
+    let reviewSort = wrapper.find('option');
+    expect(reviewSort).toHaveLength(3);
+    expect(reviewSort.at(0).text()).toBe('relevance');
+    expect(reviewSort.at(1).text()).toBe('helpful');
+    expect(reviewSort.at(2).text()).toBe('latest');
+  });
+
+  test('Sort options should have Helpful, latest, Relevance, By default, the option should be in order of relevance', () => {
+    let filters = [];
+    let wrapper = shallow(<ReviewSort onChange={() => { }} />);
+    let select = wrapper.find('select');
+    expect(select.find('option').at(0).text()).toBe('relevance');
+    expect(select.find('option').at(0).props().selected).toBe('selected');
+  });
+
+  test('Can sort helpful properly', () => {
+    let filters = [];
+    const wrapper = mount(<Review data={reviewData} />);
+    let sortedReviews = reviewData.reviews.results.slice();
+    sortedReviews.sort((a, b) => {
+      return b.helpfulness - a.helpfulness;
+    });
+    let select = wrapper.find('#review-sort-select');
+    select.find('option').at(0).instance().selected = false;
+    select.find('option').at(1).instance().selected = true;
+    select.find('option').at(2).instance().selected = false;
+    select.simulate('change');
+    let moreReviewBtn = wrapper.find('#review-morereview-btn');
+    moreReviewBtn.simulate('click');
+    moreReviewBtn = wrapper.find('#review-morereview-btn');
+    moreReviewBtn.simulate('click');
+    moreReviewBtn = wrapper.find('#review-morereview-btn');
+    let reviewBodies = wrapper.find('.review-body');
+    expect(reviewBodies.at(0).text()).toBe(sortedReviews[0].body);
+    expect(reviewBodies.at(1).text()).toBe(sortedReviews[1].body);
+    expect(reviewBodies.at(2).text()).toBe(sortedReviews[2].body);
+    expect(reviewBodies.at(3).text()).toBe(sortedReviews[3].body);
+    expect(reviewBodies.at(4).text()).toBe(sortedReviews[4].body);
+  });
+
+
+  test('Can sort latest properly', () => {
+    let filters = [];
+    const wrapper = mount(<Review data={reviewData} />);
```

Comment:
I think I remember seeing this in an earlier commit, but can't see it now since you haven't edited it so I just want to make sure.  You are using a cleanup after every test since you are using mount?

Reply:
cleanup is the function from react test lib, so it is actually to cleanup "render" from @testing-library/react. You can see, in my test cases, I used two testing frameworks: react test lib and enzyme.